api/docgen/docgen.go

```diff
@@ -88,7 +87,7 @@ func init() {
 	addExample(pid)
 	addExample(&pid)
 
-	multistoreIDExample := multistore.StoreID(50)
+	storeIDExample := uint64(50)
```

Comment:
This isn't needed if it's not its own type

Reply:
Removed.